src/xo-app/index.js

```diff
@@ -37,6 +38,18 @@ import Vm from './vm'
 export default class XoApp extends Component {
   componentDidMount () {
     this.refs.bodyWrapper.style.minHeight = this.refs.menu.getWrappedInstance().height + 'px'
+    const previousDisclaimer = cookies.get('previousDisclaimer')
```

Comment:
I think it would make sense to move this code in its own function and call it from here.